Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+
+        public static void Demo()
+        {
+
+            EnterUserData();
+            EnterUserData();
+
+        }
+
+
+        public static void EnterUserData()
+        {
+
+            string name = Lesson3.PromptFullName("Full name: ");
+            Console.WriteLine();
+
+            int age = Lesson3.PromptAge("Age: ");
+            Console.WriteLine();
+
+            float weight = Lesson3.PromptWeight("Weight: ");
+            Console.WriteLine();
+
+            float height = Lesson3.PromptHeight("Height: ");
+            Console.WriteLine();
```

Comment:
The thing with those function names is that you run into some sort of trouble.
You describe what you want to do with the data but not what happens inside the function. This is why you have two identical functions PromptWeight and PromptHeight.

Reply:
Oh, right. That makes much more sense now, I think XD So, _for example_, naming a function **PromptUserInt** or maybe even **UserDataInt** for any time we want to have a user input an integer and return it, age, weight, birth year, etc.., rather than use a function that says _PromptHeight_ for all of them.